db/migrate/20231030160545_remove_reference_from_movie_actors.rb

```diff
@@ -0,0 +1,8 @@
+class RemoveReferenceFromMovieActors < ActiveRecord::Migration[7.0]
+  def change
+    remove_reference :movie_actors, :movies, null: false, foreign_key: true
```

Comment:
As we discussed today - for an IC, it's okay to:
1. run `rails db:drop`
2. delete the offending db migration(s)
3. recreate a db migration
4. Run `rails db:{create,migrate}`

But you did a great job "fixing forward" in this case.